tests/parm/cpld_control.nml.IN

```diff
@@ -259,7 +262,18 @@
   knob_ugwp_dokdis   = @[KNOB_UGWP_DOKDIS]
   knob_ugwp_ndx4lh   = @[KNOB_UGWP_NDX4LH]
   knob_ugwp_version  = @[KNOB_UGWP_VERSION]
-  launch_level      = 54
+  ! This is only for UGWP v0
+@[HIDE_UGWPV0] launch_level      = 54
```

Comment:
I feel this brings the mismatched namelist issue up to the top namelist templates. The launch_level is used by both ugwpv0 and ugwpv1, it's confusing that it only shows up as a namelist list for ugwpv0 but is a hardcoded variable in UGWPV1. If ugwpv0 and ugwpv1 defines their own namelist with different variables, they should give namelist a different name. I'd suggest not bringing these discrepancy to top namelist template. Do we need to create an issue in CCPP physics?

Reply:
> Do we need to create an issue in CCPP physics?

Please open an issue. Renaming the namelists would clean things up.

For now, the new variables I've added are the best solution. It prevents duplication of nml files; that's always error-prone.

Reply:
Jun has opened an issue, so I'm considering this conversation resolved.

Reply:
@mdtoyNOAA FYI